client/curriculum/html/AboutHtml.tsx

```diff
@@ -0,0 +1,335 @@
+        </LessonCover>
+        <p>
+          La baza oricarei aplicatii web stau, de obicei, trei limbaje:
+          {' '}
+          <strong>HTML</strong>
+          {', '}
+          <strong>CSS</strong>
+          {' si '}
+          <strong>Javascript</strong>
+          . Probabil va intrebati de ce e nevoie de toate.
+        </p>
+        <p>
+          Raspunsul este ca fiecare are propriul scop bine definit.
+        </p>
+        <section>
+          <LessonHeading as="h3" id="analogie">
+            O simpla analogie
+          </LessonHeading>
+          <p>
+            Pentru o intelegere mai buna, ne putem gandi la constructia unei case.
+          </p>
+          <p>
+            Fundatia, peretii si acoperisul fac parte din structura de baza a
+            casei, fara de care nu putem ajunge la un rezultat final. Corespondentul
+            acestei structuri in cazul unei aplicatii web este
+            {' '}
+            <em>HTML</em>
+            . Asa cum o casa incepe cu fundatia, constructia unei aplicatii web incepe
+            cu HTML.
+          </p>
+          <p>
+            Dupa ce structura de baza este gata, urmeaza sa definim culorile folosite,
+            plasamentul geamurilor si a usilor si alte lucruri ce tin de design.
+            Pentru un programator front-end, acesta este pasul in care se introduce
+            {' '}
+            <em>CSS</em>
+            -ul.
+          </p>
+          <p>
+            La final, trebuie ca geamurile, usile si becurile puse la pasul anterior sa
+            functioneze corespunzator si sa putem interactiona cu ele. Asa cum probabil
+            ati ghicit, in cazul unei aplicatii web, aceste operatii ii revin
+            {' '}
+            <em>Javascript</em>
+            -ului.
+          </p>
+          <p>
+            Noi ne vom concentra astazi pe primul limbaj dintre cele trei enumerate
+            mai sus, si anume HTML.
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="intro">
+            Ce este HTML?
+          </LessonHeading>
+          <p>
+            <strong>HTML (Hyper Text Markup Language)</strong>
+            {' '}
+            este limbajul standard de markup folosit pentru a defini
+            structura de baza a unei pagini web. Nu este un limbaj de
+            programare propriu-zis, ci doar un limbaj descriptiv, bazat pe
+            {' '}
+            <em>XML</em>
+            .
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="elemente">
+            Elemente
+          </LessonHeading>
+          <p>
+            Orice fisier HTML este construit din elemente, numite si
+            <strong>tags</strong>
+            . Ele sunt formate din perechi de forma
+            {' '}
+            <strong className="formatted">{'<element> </element>'}</strong>
+            . Primul se numeste
+            {' '}
+            <strong>opening tag</strong>
```

Comment:
Let's use the romanian naming, and in paranthesis we can add the english one.

"tag de deschidre (opening tag)".